README.md

```diff
@@ -1,2 +1,62 @@
-# prestashop
-Prestashop template for Platform.sh. 
+# Prestashop for Platform.sh
+
+<p align="center">
+<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/prestashop/.platform.template.yaml&utm_content=prestashop&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
+    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
+</a>
+</p>
+
+This template demonstrates building a Spring Boot application. It uses Gradle to build the application and deploy it to Platform.sh.
+
+This template builds Prestashop project.  It is pre-configured to use MariaDB.
+
+PrestaShop is a freemium, open source e-commerce solution. The software is published under the Open Software License. It is written in the PHP programming language with support for the MySQL database management system. PrestaShop is currently used by 300,000 shops worldwide and is available in 60 different languages.
+
+
+## Features
+
+* PHP 7.4
+* MariaDB 10.4
+* Automatic TLS certificates
```

Comment:
No composer-build?  Does it not use Composer?

Reply:
Mention the Composer-based build, assuming that's still the case.  We do for other PHP apps.